backend/backend-001/src/main/java/com/puggysoft/tools/salary/SqlSalaryEmployeeFilterBuilderNative.java

```diff
@@ -0,0 +1,27 @@
+package com.puggysoft.tools.salary;
+
+import com.puggysoft.dtos.salary.DtoSalaryEmployeeFilter;
+import com.puggysoft.tools.SqlFilterBuilder;
+
+public final class SqlSalaryEmployeeFilterBuilderNative {
+
+  private SqlSalaryEmployeeFilterBuilderNative(){}
+
+  /**
+   * Method to build the filter salary employee query.
+   * @return the salary employee query.
+   */
+  public static String build(DtoSalaryEmployeeFilter dtoSalaryEmployeeFilter) {
+    String query = "";
+    query = query + SqlFilterBuilder.getFilterQuery("empleado", dtoSalaryEmployeeFilter.empleadoCriteria, dtoSalaryEmployeeFilter.empleadoOperator);
```

Comment:
add table name here please. 
In the future when we use this filter together with joins it can cause conflicts if we don't put the name of the table.
Eg. sueldos_detalle_contratacion.cargo

Reply:
Done.